### PR TITLE
Android testing with GMD

### PR DESCRIPTION
**Pull request by yschimke**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -409,35 +409,20 @@ jobs:
...
-        with:
-          api-level: ${{ matrix.api-level }}
-          force-avd-creation: false
-          emulator-options: -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
-          disable-animations: false
-          script: echo "Generated AVD snapshot for caching."
+      - name: Accept licenses
+        run: |
+          mkdir -p /Users/runner/Library/Android/sdk
+          tar xf .github/android_licenses.tar.gz -C /Users/runner/Library/Android/sdk
```
Comment (swankjesse) [inline .github/workflows/build.yml:415]: This song and dance works for me, and it’s also bizarre.

Who is accepting these licenses? Square, Inc.? Does accepting it both mechanically and publicly obligate us to anything?

Comment (swankjesse) [reply .github/workflows/build.yml:415]: I asked some internal people if we have Opinions on stuff like this.

Comment (yschimke) [reply .github/workflows/build.yml:415]: When you run various gradle plugins, it writes these files, it just doesn't know about previews

Comment (yschimke) [reply .github/workflows/build.yml:415]: I guess I should be using a step like the following with channel set.  Will test.

```
yes | "$ANDROID_HOME"/cmdline-tools/latest/bin/sdkmanager --licenses || true
```

**Comment by yschimke** [comment]
I think we'll need fixes from GMD in 8.4? so maybe to wait for this.

**Pull request closed without merging**